### PR TITLE
[release/6.0] Update application-insights to 2.19.0

### PR DESCRIPTION
**Pull request by lbussell**
This should fix https://github.com/dotnet/source-build/issues/2994.

Also see
* https://github.com/dotnet/sdk/pull/22974

### CODE REVIEW

**Review by MichaelSimons — APPROVED** [review]

**Comment by MichaelSimons** [comment]
@lbussell - Are these changes also needed?  https://github.com/dotnet/source-build-externals/pull/33

**Comment by lbussell** [comment]
> Are these changes also needed?

Trying to figure that out now - will add them if needed.

**Comment by MichaelSimons** [comment]
IIRC this is only an issue in the tarball where it removes the online feeds from the known nuget.configs.

**Comment by MichaelSimons** [comment]
...The result is the potential for prebuilts because NuGet non-determinism.

File: eng/SourceBuildPrebuiltBaseline.xml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <UsageData>
   <IgnorePatterns>
-    <UsagePattern IdentityGlob="*/*" />
+    <UsagePattern IdentityGlob="Microsoft.SourceBuild.Intermediate.*/*" />
```
Comment (MichaelSimons) [inline eng/SourceBuildPrebuiltBaseline.xml:3]: I don't think we should be enabling prebuilt detection with these changes.  There are likely other changes needed to get the repo level prebuilts clean.  We don't want to block this PR on them.

**Pull request merged**